public/app/features/dashboard/state/initDashboard.ts

```diff
@@ -102,6 +103,29 @@ async function fetchDashboard(
   }
 }
 
+const logQueryOnDashboardPanels = (panels: PanelModel[], dashboardId: string, collapsed?: boolean): void => {
+  panels.forEach((panel) => {
+    // panel is collapsed or parent is
+    collapsed = !!panel.collapsed || !!collapsed;
+    if (panel.panels) {
+      logQueryOnDashboardPanels(panel.panels, dashboardId, collapsed);
+    } else {
+      panel.targets.forEach((target) => {
+        reportInteraction('grafana_dashboard_loaded', {
+          dashboard_id: dashboardId,
+          panel_id: panel.id,
+          panel_type: panel.type,
```

Comment:
Is it possible to add these information as well?

the http status code
user id, or some id that will later on help us count distinct users
trace id for correlation, the same trace id that's used on other contexts
self-identifying information like the url of this grafana instance.
For the http status code, I cannot tell whether this logging happens before or after the query has completed. If it's the former, I can understand we won't have that status code.

Reply:
Hi @brahmnes , I have taken note of the issue you raised regarding telemetry (https://github.com/grafana/grafana-enterprise-mirror-shared-microsoft/issues/78). 
I will see what I can easily add to this event, but it might be that we are looking at two different events here; We want to emit as few events as possible in this PR so we will only log a list of queries for Azure Monitor datasources, that means we won't be logging query-specific data (traceId / http code). 
We will look into query logs as per your issue, but since it's a different question you are trying to answer, we might implement another solution for them (possibly backend logs).

_Note: This PR is meant as a starting point (MVP) for a frontend event, the idea being introducing a first iteration of adoption metrics for our datasources. More iterations will come._

Reply:
Hi @yaelleC , thank you for addressing the comment.
I look forward to the solution you come up with.